migrations.go

```diff
@@ -53,14 +55,40 @@ func migrate_4_SwarmKey(cfg *Config) bool {
 // for bootstrap nodes.
 // Replaces all bootstrap nodes with default values if so.
 func migrate_5_Bootstrap_node(cfg *Config) bool {
-	obsoleteBootstrapNodeList := []string{"3.120.224.94"}
+	obns := []string{
+		"3.120.224.94",
+	}
+	peers, _ := DefaultBootstrapPeers()
+	return doMigrateNodees(cfg, obns, peers)
```

Comment:
spelling correction doMigrateNodes instead of doMigrateNodees?

Reply:
ah, my keyboard